### ISSUE TITLE
eix parses package.accept_keywords :0 but not :0/0

### ISSUE THREAD

**Issue by edgar444** [vaeth/eix#70]
I was playing around with understanding slots and sub-slots, and noticed this oddity. Where `:0` produces `(~)1.4.0`, whereas `:0/0` produces `~1.4.0`, yet should be `(~)1.4.0` too.

Without accept keyword entry:

```
ed@Blackout-Gentoo ~ $ grep gevent /etc/portage/package.accept_keywords/maybe
1 ed@Blackout-Gentoo ~ $ emerge -pvq dev-python/gevent
[ebuild     U ] dev-python/gevent-1.3.7 [1.2.1] USE="doc* -examples" PYTHON_TARGETS="python3_6 -python2_7* (-python3_7) (-pypy%) (-python3_5%)"

ed@Blackout-Gentoo ~ $ eix -e gevent
[U] dev-python/gevent
     Available versions:  1.0.1^t 1.2.1^t ~1.2.2^t ~1.3.5^t 1.3.7^t ~1.4.0^t {doc examples PYTHON_TARGETS="python2_7 python3_6 python3_7 python3_8"}
     Installed versions:  1.2.1^t(01:40:34 PM 08/09/2019)(-doc -examples PYTHON_TARGETS="python2_7 python3_6 -pypy -python3_5")
     Homepage:            http://gevent.org/ https://pypi.org/project/gevent/
     Description:         Coroutine-based network library

ed@Blackout-Gentoo ~ $
```

Works with slot specifier:

```
ed@Blackout-Gentoo ~ $ grep gevent /etc/portage/package.accept_keywords/maybe
=dev-python/gevent-1.4.0:0::gentoo ~amd64
ed@Blackout-Gentoo ~ $ emerge -pvq dev-python/gevent
[ebuild     U ] dev-python/gevent-1.4.0 [1.2.1] USE="doc* -examples" PYTHON_TARGETS="python3_6 -python2_7* -python3_7% -python3_8% (-pypy%) (-python3_5%)"

ed@Blackout-Gentoo ~ $ eix -e gevent
[U] dev-python/gevent
     Available versions:  1.0.1^t 1.2.1^t ~1.2.2^t ~1.3.5^t 1.3.7^t (~)1.4.0^t {doc examples PYTHON_TARGETS="python2_7 python3_6 python3_7 python3_8"}
     Installed versions:  1.2.1^t(01:40:34 PM 08/09/2019)(-doc -examples PYTHON_TARGETS="python2_7 python3_6 -pypy -python3_5")
     Homepage:            http://gevent.org/ https://pypi.org/project/gevent/
     Description:         Coroutine-based network library

ed@Blackout-Gentoo ~ $
```

Doesn't work with subslot specifier:

```
ed@Blackout-Gentoo ~ $ grep gevent /etc/portage/package.accept_keywords/maybe
=dev-python/gevent-1.4.0:0/0::gentoo ~amd64
ed@Blackout-Gentoo ~ $ emerge -pvq dev-python/gevent
[ebuild     U ] dev-python/gevent-1.4.0 [1.2.1] USE="doc* -examples" PYTHON_TARGETS="python3_6 -python2_7* -python3_7% -python3_8% (-pypy%) (-python3_5%)"

ed@Blackout-Gentoo ~ $ eix -e gevent
[U] dev-python/gevent
     Available versions:  1.0.1^t 1.2.1^t ~1.2.2^t ~1.3.5^t 1.3.7^t ~1.4.0^t {doc examples PYTHON_TARGETS="python2_7 python3_6 python3_7 python3_8"}
     Installed versions:  1.2.1^t(01:40:34 PM 08/09/2019)(-doc -examples PYTHON_TARGETS="python2_7 python3_6 -pypy -python3_5")
     Homepage:            http://gevent.org/ https://pypi.org/project/gevent/
     Description:         Coroutine-based network library

ed@Blackout-Gentoo ~ $
```

Eix version is 0.33.9.

**Comment by edgar444** [comment]
The eix can parse `:0/2.0.6` though! So unique to subslots `:0/0`

Without the `:0/2.0.6` subslot:

```
ed@Blackout-Gentoo ~ $ grep 'ounit.*2.0.6' /etc/portage/package.accept_keywords/forced
1 ed@Blackout-Gentoo ~ $ eix -e ounit
[I] dev-ml/ounit
     Available versions:  2.0.0(0/2.0.0) (~)2.0.5(0/2.0.5) ~2.0.6(0/2.0.6) {debug doc +ocamlopt}
     Installed versions:  2.0.5(0/2.0.5)(09:41:18 PM 03/05/2020)(doc ocamlopt -debug)
     Homepage:            http://ounit.forge.ocamlcore.org/
     Description:         Unit testing framework for OCaml

ed@Blackout-Gentoo ~ $
```

With the `:0/2.0.6` subslot:

```
ed@Blackout-Gentoo ~ $ grep 'ounit.*2.0.6' /etc/portage/package.accept_keywords/forced
=dev-ml/ounit-2.0.6:0/2.0.6::gentoo ~amd64
ed@Blackout-Gentoo ~ $ eix -e ounit
[U] dev-ml/ounit
     Available versions:  2.0.0(0/2.0.0) (~)2.0.5(0/2.0.5) (~)2.0.6(0/2.0.6) {debug doc +ocamlopt}
     Installed versions:  2.0.5(0/2.0.5)(09:41:18 PM 03/05/2020)(doc ocamlopt -debug)
     Homepage:            http://ounit.forge.ocamlcore.org/
     Description:         Unit testing framework for OCaml

ed@Blackout-Gentoo ~ $
```

**Comment by vaeth** [comment]
What is actually strange is portage (or pms) wanting to interpret "/0" and having no subslot equally.
However, since this is apparently portage's behavior, I adapt eix correspondingly.
This is fixed in eix master (>=0.33.12_alpha).
